### PR TITLE
Adds support for the newest Bridge version and refactored UI/UX.

### PR DESCRIPTION
**Pull request by aguschirico**
This pull request includes:
- UI Reskinning: Changes requested by Product for better UX and a renewed UI.
- Support for Bridge v4: Matches functionality added to the bridge over the last months, like BSC support and NFT Bridge (WIP).
- Refactor on validations, component hierarchy and styles
- Added webpack and vuetify support
- Updated dependencies

### CODE REVIEW

File: src/components/HelloWorld.vue [hunk 1]
```diff
@@ -0,0 +1,133 @@
+<template>
+  <v-container>
+    <v-row class="text-center">
+      <v-col cols="12">
+        <v-img :src="logo" class="my-3" contain height="200" />
+      </v-col>
+
+      <v-col class="mb-4">
+        <h1 class="display-2 font-weight-bold mb-3">
+          <div>Welcome to the Vuetify 3 Alpha</div>
```
Comment (pmprete) [inline src/components/HelloWorld.vue:10]: We can remove the HelloWorld page

File: src/components/crossForm/CrossForm.vue [hunk 2]
```diff
@@ -1,5 +1,12 @@
 <template>
-  <Form id="crossForm" ref="crossForm" name="crossForm" class="cross-form" @submit="onSubmit">
+  <Form
+    v-if="false"
```
Comment (pmprete) [inline src/components/crossForm/CrossForm.vue:3]: no need of v-if if it's never gonna be true

File: src/components/crossForm/CrossForm.vue [hunk 3]
```diff
@@ -66,7 +73,7 @@
               class="outline form-control text-center"
               :class="{ disabled: disabled }"
               placeholder="Amount"
-              :disabled="disabled"
+              :disabled="disabled || disabledAmount"
```
Comment (pmprete) [inline src/components/crossForm/CrossForm.vue:76]: not sure if disabled make sense at this point. If so change
```js
 :class="{ disabled: disabled }"
 ```
 To follow the same rule as :disabled

File: src/components/crossForm/CrossForm.vue [hunk 4]
```diff
@@ -400,64 +533,137 @@ export default {
...
+              },
+              rpcUrls: [networkConfig.rpc],
+            },
+          ],
+        })
+      } catch (error) {
+        console.error(error)
+      }
+    },
+    async handleSwitchNetwork() {
```
Comment (pmprete) [inline src/components/crossForm/CrossForm.vue:590]: This method looks too similar to handleChangeNetwork, maybe we should use only one of them

File: src/components/crossForm/CrossForm.vue [hunk 5]
```diff
@@ -400,64 +533,137 @@ export default {
...
-    },
-    refreshBalanceAndAllowance() {
-      const data = this
-      const web3 = data.sharedState.web3
-      const config = data.sharedState.currentConfig
-      const token = data.selectedToken
+      this.selectedToken = {};
+      this.willReceiveToken = null;
+      this.amount = 0
+      // this.$refs.crossForm.resetForm()
```
Comment (pmprete) [inline src/components/crossForm/CrossForm.vue:628]: reset form sounds like it should be uncommented, is this on purpose?

File: src/components/formWrapper/FormWrapper.vue [hunk 6]
```diff
@@ -1,6 +1,6 @@
 <template>
   <div>
-    <div>
+    <div v-if="false">
```
Comment (pmprete) [inline src/components/formWrapper/FormWrapper.vue:3]: i think we don't need the v-if it has a harcoded value

File: src/components/transactions/Transactions.vue [hunk 7]
```diff
@@ -1,5 +1,5 @@
 <template>
-  <div class="transactions">
+  <div id="home" class="transactions">
```
Comment (pmprete) [inline src/components/transactions/Transactions.vue:2]: is this ok? i think id=homas has been used in other divs. Ids should be unique, if you need a selector for css we should use a class instead

**Review by pmprete — APPROVED** [review]
Except my comments looks good

File: src/components/transactions/Transactions.vue (hunk 7 above)
Comment (aguschirico) [reply src/components/transactions/Transactions.vue]: fixed

File: src/components/formWrapper/FormWrapper.vue (hunk 6 above)
Comment (aguschirico) [reply src/components/formWrapper/FormWrapper.vue:3]: removed

File: src/components/crossForm/CrossForm.vue (hunk 2 above)
Comment (aguschirico) [reply src/components/crossForm/CrossForm.vue]: removed

File: src/components/HelloWorld.vue (hunk 1 above)
Comment (aguschirico) [reply src/components/HelloWorld.vue:10]: done

File: src/components/crossForm/CrossForm.vue (hunk 3 above)
Comment (aguschirico) [reply src/components/crossForm/CrossForm.vue]: it was under the v-if="false" so it was removed.

File: src/components/crossForm/CrossForm.vue (hunk 4 above)
Comment (aguschirico) [reply src/components/crossForm/CrossForm.vue:371]: it's similar, yet not the same. One is for switching origin-destination, and the other is to change the origin to something else.

File: src/components/title/Title.vue [hunk 8]
```diff
@@ -33,4 +33,7 @@ export default {
   text-align: center;
   margin-bottom: 5px;
 }
+#title{
```
Comment (coolwather) [inline src/components/title/Title.vue:36]: Maybe we can create a title class to be reusable.

**Review by coolwather — APPROVED** [review]

**Review by pmprete — APPROVED** [review]
Great job!

**Pull request merged**